main.go

```diff
@@ -798,15 +880,16 @@ func DefineFlags() {
 func PrintUsage() {
 	PrintASCIIArt()
 	util.PrintUtil("\nUsage:\tseed COMMAND\n\n")
-	util.PrintUtil("A test runner for seed spec compliant algorithms\n\n")
+	util.PrintUtil("A tool for assisting in creating seed spec compliant algorithms\n\n")
 	util.PrintUtil("Commands:\n")
 	util.PrintUtil("  build \tBuilds Seed compliant Docker image\n")
 	util.PrintUtil("  batch \tExecutes Seed compliant docker image over multiple iterations\n")
 	util.PrintUtil("  init  \tInitialize new project with example seed.manifest.json file\n")
-	util.PrintUtil("  list  \tAllows for listing of all Seed compliant images residing on the local system\n")
-	util.PrintUtil("  publish\tAllows for publish of Seed compliant images to remote Docker registry\n")
-	util.PrintUtil("  pull\t\tAllows for pulling Seed compliant images from remote Docker registry\n")
-	util.PrintUtil("  run   \tExecutes Seed compliant Docker docker image\n")
+	util.PrintUtil("  list  \tLists all Seed compliant images residing on the local system\n")
+	util.PrintUtil("  publish\tPublishes Seed compliant images to remote Docker registry\n")
+	util.PrintUtil("  unpublish\tRemoves images from remote Docker registry\n")
```

Comment:
We had these commands listed in alphabetical order. I'd suggest moving unpublish to the bottom to maintain that order.